interfaces/HO-Portal/src/app/iframe/recipient.page.html

```diff
@@ -1,5 +1,39 @@
-<ion-content class="ion-padding">
-  <app-recipient-details></app-recipient-details>
-  <app-recipient-details></app-recipient-details>
-  <app-recipient-details></app-recipient-details>
+<ion-content>
+  <ng-container *ngIf="recipients?.length == 0 && queryParamPhonenumber">
+    <div class="ion-padding">
+      <h4>{{"page.iframe.recipient.no-recipients-found" | translate}}</h4>
+    </div>
+  </ng-container>
+  <ng-container *ngIf="!queryParamPhonenumber">
+    <div class="ion-padding">
+      <h4>{{"page.iframe.recipient.no-phone-number" | translate}}</h4>
+    </div>
+  </ng-container>
+  <ng-container *ngIf="recipients && programsMap">
+    <ion-accordion-group [value]="accordionGroupValue">
+      <ion-accordion
+        *ngFor="let recipient of recipients"
+        value="{{recipient.id}}"
```

Comment:
```suggestion
        [value]="recipient.id"
```

👆 Would be the correct syntax.

I also think that Prettier has to take a good look at this file. ;)